### PR TITLE
Adding code of conduct

### PR DESCRIPTION
**Pull request by juliendonck**
Hey, took the code of conduct @angedupre and @hxrts worked on and added it to the repo. Would like to link to it from the website.

To review: 
- link from readme.md
- email address in coc

### CODE REVIEW

**Comment by juliendonck** [comment]
ping @hxrts wanna merge this today

**Comment by hxrts** [comment]
this LGTM, though not entirely sure about the email address. @cloudhead mentioned @geigerzaehler was looking into this. I think we ultimately want *@radicle.xyz to forward to a monadic.xyz address.

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Code of Conduct
+
+We, the Radicle maintainers, intend to create a developer community based on common principles of mutual respect, inclusivity, and collaboration with the presumption of good faith. This code of conduct was drafted as a public commitment to a shared social contract.
+
+By engaging with the Radicle repository you agree to these standards of exchange that we consider necessary to ensure the health of the Radicle community overall.
+
+We consider this a living document that should be updated according to the needs of the Radicle developer community. We welcome comments and proposals for future iterations and invite you to visit the git history if you are interested its evolution.
```
Comment (MeBrei) [inline code-of-conduct.md]: interested `in` its evolution ?

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+To get the full value of such feedback, recipients should stay open minded and assume contributors are acting in the best interest of the project. Feedback meeting the standard above should not be interpreted as antagonistic.
+Communication should always come from a place of respect for other contributors. Bullying, vulgarity, intolerance, or alienation based on gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics do not meet our standard for respect.
+
+Comments that denigrate an individual’s identity create a climate of fear that drives contributors to exit, and diminishing this community does not serve our shared purpose.
+
+Should you feel certain behaviour constitutes a transgression of our community standards or that community members have been placed at risk, please email <radicle@monadic.xyz>. Project maintainers are responsible for addressing these concerns will use their discretion to secure the health of the Radicle developer community.
```
Comment (MeBrei) [inline code-of-conduct.md]: Project maintainers are responsible for addressing these concerns `and` will use their discretion to secure the health of the Radicle developer community.

OR

Project maintainers`,` responsible for addressing these concerns`,` will use their discretion to secure the health of the Radicle developer community.

Comment (juliendonck) [reply code-of-conduct.md]: nice catch, updated it

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+To get the full value of such feedback, recipients should stay open minded and assume contributors are acting in the best interest of the project. Feedback meeting the standard above should not be interpreted as antagonistic.
+Communication should always come from a place of respect for other contributors. Bullying, vulgarity, intolerance, or alienation based on gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics do not meet our standard for respect.
+
+Comments that denigrate an individual’s identity create a climate of fear that drives contributors to exit, and diminishing this community does not serve our shared purpose.
+
+Should you feel certain behaviour constitutes a transgression of our community standards or that community members have been placed at risk, please email <radicle@monadic.xyz>. Project maintainers are responsible for addressing these concerns and will use their discretion to secure the health of the Radicle developer community.
```
Comment (jkarni) [inline code-of-conduct.md]: This is the mailing list! Not a good place to email about these things, I think (and at the very least we have to make clear these email would be public)

**Pull request merged**